src/Symfony/Component/HttpKernel/EventListener/ErrorListener.php

```diff
@@ -104,9 +99,10 @@ public function onKernelException(ExceptionEvent $event)
         try {
             $response = $event->getKernel()->handle($request, HttpKernelInterface::SUB_REQUEST, false);
         } catch (\Exception $e) {
+            $logLevel = $this->resolveLogLevel($e);
```

Comment:
`logException()` already resolves the log level for us, doesn't it. This call feels redundant.